app/build.gradle

```diff
@@ -46,8 +46,8 @@ android {
             jniDebuggable true
         }
         debug {
-            buildConfigField 'String', "MOCK_USERNAME", properties.getProperty('MOCK_USERNAME', '"xxxxx"')
-            buildConfigField 'String', "MOCK_PASSWORD", properties.getProperty('MOCK_PASSWORD', '"xxxxx"')
+            buildConfigField 'String', "MOCK_USERNAME", properties.getProperty('MOCK_USERNAME', '"isaac55577"')
```

Comment:
please remove your credentials and regenerate passoword

Reply:
Sure on it, also deleted the connection.